sympy/ntheory/factor_.py

```diff
@@ -2201,3 +2201,29 @@ def is_deficient(n):
     if is_perfect(n):
         return False
     return bool(abundance(n) < 0)
+
+
+def is_amicable(m, n):
+    """Returns True if the pair ``(m, n)`` is a Amicable Number, else False.
+
+    Amicable numbers are two different numbers so related that the sum of the proper divisors of each is equal to the other number.
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory.factor_ import is_deficient
```

Comment:
import `is_amicable` rather than `is_deficient`

Reply:
but i think it should be `from sympy.ntheory.factor_ import divisor_sigma`

Reply:
@Yathartha22 but i think it should be `from sympy.ntheory.factor_ import divisor_sigma` because i have used the function `divisor_sigma`  in `is_amicable`

Reply:
or I think both `from sympy.ntheory.factor_ import divisor_sigma, is_amicable` should be imported

Reply:
@Yathartha22 thanks for pointing out